ml-agents/mlagents/trainers/tests/simple_test_envs.py

```diff
@@ -38,29 +38,33 @@ class SimpleEnvironment(BaseEnv):
     def __init__(
         self,
         brain_names,
-        use_discrete,
         step_size=STEP_SIZE,
         num_visual=0,
         num_vector=1,
         vis_obs_size=VIS_OBS_SIZE,
         vec_obs_size=OBS_SIZE,
-        action_size=1,
+        action_sizes=(1, 0),
     ):
         super().__init__()
-        self.discrete = use_discrete
         self.num_visual = num_visual
         self.num_vector = num_vector
         self.vis_obs_size = vis_obs_size
         self.vec_obs_size = vec_obs_size
-        if use_discrete:
-            action_spec = ActionSpec.create_discrete(
-                tuple(2 for _ in range(action_size))
-            )
+        continuous_action_size, discrete_action_size = action_sizes
+        discrete_tuple = tuple(2 for _ in range(discrete_action_size))
+        if continuous_action_size > 0:
+            if discrete_action_size > 0:
+                action_spec = ActionSpec(continuous_action_size, discrete_tuple)
```

Comment:
I should be possible to construct an ActionSpec with `discrete_tuple = ()` or `continuous_action_size=0`. This should simplify this code.

Reply:
It was explicitly requested in the ActionSpec PR by @chriselion to have static methods that would create ActionSpecs in the continuous/discrete case. Also, the ActionSpec would handle that without issue.

Reply:
I removed it.